rest/util.py

```diff
@@ -60,3 +61,19 @@ def dispatch(self, request, *args, **kwargs):
             CheckValidMemberParamMixin,
             self
         ).dispatch(request, *args, **kwargs)
+
+
+def default_slugify(label, default_name, exists_func):
```

Comment:
the reason was because it looked weid to me to have a term and a repository have the same base name.